### ISSUE TITLE
Release `1.30.2` contains post-v1 module path in the `go.mod` file

### ISSUE THREAD

**Issue by pbusko** [buildpacks/libcnb#286]
Seems like the latest `v1` release got released using the `main` branch instead of `release-1.x`

```bash
$ go get github.com/buildpacks/libcnb@v1.30.2
go: github.com/buildpacks/libcnb@v1.30.2: invalid version: go.mod has post-v1 module path "github.com/buildpacks/libcnb/v2" at revision v1.30.2
```

**Comment by dmikusa** [comment]
My fault. I don't know what I was doing here 🤦 

I cut a release for 1.x from the main branch, which is obviously wrong. I'll mark that one as unusable and cut a release the correct way here shortly. Look for 1.30.3 soon.

**Comment by dmikusa** [comment]
Ok, 1.30.3 has been released and I've put a note on 1.30.2 telling people to not use it.

```
> go get -u github.com/buildpacks/libcnb@v1.30.3
go: downloading github.com/buildpacks/libcnb v1.30.3
go: downloading github.com/onsi/gomega v1.33.0
go: upgraded github.com/buildpacks/libcnb v1.30.1 => v1.30.3
go: upgraded github.com/onsi/gomega v1.32.0 => v1.33.0

> go list -m --versions github.com/buildpacks/libcnb
github.com/buildpacks/libcnb v1.0.0 v1.1.0 v1.1.1 v1.1.2 v1.2.0 v1.3.0 v1.4.0 v1.4.1 v1.4.2 v1.5.0 v1.6.0 v1.6.1 v1.7.0 v1.8.0 v1.9.0 v1.10.0 v1.10.1 v1.11.0 v1.12.0 v1.12.1 v1.13.0 v1.14.0 v1.14.1 v1.15.0 v1.15.1 v1.15.2 v1.16.0 v1.17.0 v1.18.0 v1.18.1 v1.19.0 v1.20.0 v1.21.0 v1.22.0 v1.23.0 v1.24.0 v1.25.0 v1.25.1 v1.25.2 v1.25.3 v1.25.4 v1.25.5 v1.26.0 v1.27.0 v1.28.0 v1.29.0 v1.30.0 v1.30.1 v1.30.2 v1.30.3
```

Please let me know if you're still seeing any issues & again, sorry about that.
